examples/features/customloadbalancer/client/customroundrobin/customroundrobin.go

```diff
@@ -88,15 +88,18 @@ type customRoundRobin struct {
 	balancer.ClientConn
 	bOpts balancer.BuildOptions
 
-	cfg *customRRConfig
+	cfg atomic.Pointer[customRRConfig]
 }
 
 func (crr *customRoundRobin) UpdateClientConnState(state balancer.ClientConnState) error {
 	crrCfg, ok := state.BalancerConfig.(*customRRConfig)
 	if !ok {
 		return balancer.ErrBadResolverState
 	}
-	crr.cfg = crrCfg
+	if el := len(state.ResolverState.Endpoints); el != 2 {
```

Comment:
Nit: slightly nicer is `if el := state.ResolverState.Endpoints; len(el) != 2 { ... }` then the error will include the actual list instead of just the size.

Reply:
Good point. Done.